<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="study_track_gate_block">
    <static>true</static>

    <link name="gate">
      <pose>0 0 0 0 0 3.1416</pose>
      <collision name="collision">
        <geometry>
          <mesh>
            <uri>model://study_track_gate_block/meshes/gate.stl</uri>
            <scale>1 1 1</scale>
          </mesh>
        </geometry>
      </collision>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://study_track_gate_block/meshes/gate.stl</uri>
            <scale>1 1 1</scale>
          </mesh>
        </geometry>
        <material>
          <ambient>0.7 0.7 0.7 1</ambient>
          <diffuse>0.8 0.8 0.8 1</diffuse>
          <specular>0.1 0.1 0.1 1</specular>
          <emissive>0 0 0 0</emissive>
        </material>
      </visual>
    </link>
    
  </model>
</sdf>
